doc/user/PARSING.md

```diff
@@ -57,7 +58,15 @@ When formatting a ratio into a string, it is formatted as a double.
 
 ## String
 
-String are parsed and formatted as is.
+String are trimmed of leading and trailing space when parsed.
+String are formatted as is.
+
+## String
+
+Path are formatted by creating a [std::filesystem::path](https://en.cppreference.com/w/cpp/filesystem/path/path)
+and then collapsed using [f3d::utils::collapsePath](https://f3d.app/doc/libf3d/doxygen/api/classf3d_1_1utils.html#_CPPv4N3f3d5utils12collapsePathERKNSt10filesystem4pathERKNSt10filesystem4pathE).
```

Comment:
Im not convinced by my doc here but I also do not know if I should be super specific.